Pipfile

```diff
@@ -36,9 +36,11 @@ pycryptodomex = "*"
 
 [dev-packages]
 black = "==20.8b1"
-pytest = "*"
-pytest-cov = "*"
+mock = "*"
```

Comment:
Need to check whether `Pipfile.lock` gets recreated when installing with `Pipenv`. Test on a fresh instance.

Reply:
it does, if you delete it before

Reply:
If it's not "up to date" it will modify it. Is the Pipfile.lock a fresh one? I usually build them on a fresh `multipass` instance... but I'm sure there are better ways.